es2018.js

```diff
@@ -152,6 +159,62 @@ var ES2018 = assign(assign({}, ES2017), {
 		return target;
 	},
 
+	// https://ecma-international.org/ecma-262/9.0/#sec-getiterator
+	GetIterator: function GetIterator(obj, hint, method) {
+		var actualHint = hint;
+		if (arguments.length < 2) {
+			actualHint = 'sync';
+		}
+		if (actualHint !== 'sync' && actualHint !== 'async') {
+			throw new $TypeError("Assertion failed: `hint` must be one of 'sync' or 'async', got " + inspect(hint));
+		}
+		var actualMethod = method;
+		if (arguments.length < 3) {
+			if (actualHint === 'async') {
+				if (hasSymbols && $asyncIterator) {
+					actualMethod = this.GetMethod(obj, $asyncIterator);
+				}
+				if (actualMethod === undefined) {
+					throw new $TypeError("async from sync iterators aren't currently supported");
+				}
+			} else {
+				actualMethod = getIteratorMethod(this, obj);
+			}
+		}
+
+		var iterator = this.Call(actualMethod, obj);
+		if (this.Type(iterator) !== 'Object') {
+			throw new $TypeError('iterator must return an object');
+		}
+
+		return iterator;
+
+		// TODO: This should return an IteratorRecord
+		/*
+		var nextMethod = this.GetV(iterator, 'next');
+		return {
+			'[[Iterator]]': iterator,
+			'[[NextMethod]]': nextMethod,
+			'[[Done]]': false
+		};
+		*/
```

Comment:
This is also a breaking change, which should probably be made as part of the v2.0 release, which should be where this gets first released.

Reply:
I’m doing this in #68.